docs/cloud-native-security/session-view.asciidoc

```diff
@@ -46,7 +46,8 @@ fields collected when this setting is enabled, refer to the https://github.com/e
 [float]
 [[open-session-view]]
 === Open Session View
-Session View is accessible from the **Hosts**, **Alerts**, and **Timelines** pages, as well as the **Kubernetes** dashboard.
+Session View is accessible from the **Hosts**, **Alerts**, and **Timelines** pages, as well as the alert details flyout and the **Kubernetes** dashboard.
+//Consider linking to new alert details docs.
```

Comment:
None of the other pages are linked, so I won't link to the alert details flyout here.

```suggestion
```